addons/website/models/ir_module_module.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models
+
+
+class Module(models.Model):
+    _inherit = 'ir.module.module'
+
+    def _check(self):
+        super()._check()
+        View = self.env['ir.ui.view']
+        website_views_to_adapt = getattr(self.pool, 'website_views_to_adapt', [])
+        for module in self:
+            if website_views_to_adapt and module.name == 'website':
```

Comment:
This loop is useless.
```suggestion
if website_views_to_adapt:
```

Reply:
Indeed since I now clear the registry entry the loop can be avoided, change done